server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java

```diff
@@ -1183,7 +1183,7 @@ public void moveRuleToTheBottomTest() {
     }
 
     @Test(expected = InvalidParameterValueException.class)
-    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccomodateTheNewRuleWithOtherruleColliding() {
+    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccommodateTheNewRuleWithOtherruleColliding() {
```

Comment:
```suggestion
    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccommodateTheNewRuleWithOtherRuleColliding() {
```

Reply:
guess I'm blind but what changed in your suggestion @stephankruggg ?

Reply:
`WithOtherrule` -> `WithOtherRule`